packages/textlint/test/cli/cli-test.ts

```diff
@@ -3,7 +3,6 @@
 const assert = require("assert");
 const cli = require("../../src/index").cli;
```

Comment:
Use `import { cli } from "../../src/index"` instead of it.

Then, you can remove `: number` from `cli.execute(...).then((result: number) => { ...`.
Because, type inference is enabled by `import` statement.
(CommonJS `require` function make `cli` object `any` type)

- https://www.typescriptlang.org/docs/handbook/modules.html